src/pyOpenMS/pxds/FIAMSDataProcessor.pxd

```diff
@@ -16,14 +16,52 @@ cdef extern from "<OpenMS/ANALYSIS/ID/FIAMSDataProcessor.h>" namespace "OpenMS":
         #
         # wrap-inherits:
         #  DefaultParamHandler
-        FIAMSDataProcessor() nogil except +
-        FIAMSDataProcessor(FIAMSDataProcessor) nogil except +
+        FIAMSDataProcessor() nogil except + # wrap-doc:Data processing for FIA-MS data
+        FIAMSDataProcessor(FIAMSDataProcessor) nogil except + # wrap-ignore
         bool run(MSExperiment & experiment, float & n_seconds, MzTab & output, bool load_cached_spectrum) nogil except +
+            # wrap-doc:
+                #   Run the full analysis for the experiment for the given time interval
+                #   -----
+                #   The workflow steps are:
+                #   - the time axis of the experiment is cut to the interval from 0 to n_seconds
+                #   - the spectra are summed into one along the time axis with the bin size determined by mz and instrument resolution
+                #   - data is smoothed by applying the Savitzky-Golay filter
+                #   - peaks are picked
+                #   - the accurate mass search for all the picked peaks is performed
+                #
+                #   The intermediate summed spectra and picked peaks can be saved to the filesystem. 
+                #   Also, the results of the accurate mass search and the signal-to-noise information 
+                #   of the resulting spectrum is saved.
+                #   -----
+                #   :param experiment: Input MSExperiment
+                #   :param n_seconds: Input number of seconds
+                #   :param load_cached_spectrum: Load the cached picked spectrum if exists
+                #   :param output: [out] Output of the accurate mass search results
+                #   :return: a boolean indicating if the picked spectrum was loaded from the cached file
         # void cutForTime(MSExperiment & experiment, float & n_seconds, libcpp_vector[ MSSpectrum ] & output) nogil except +
         # NAMESPACE # MSSpectrum mergeAlongTime(libcpp_vector[ OpenMS::MSSpectrum ] & input_) nogil except +
         MSSpectrum extractPeaks(MSSpectrum & input_) nogil except +
+            # wrap-doc:
+                #   Pick peaks from the summed spectrum
+                #   -----
+                #   :param input: Input vector of spectra
+                #   :return: a spectrum with picked peaks
```

Comment:
Capital letter? Or do you think it looks better like this? Please also see the other `:return: `statements.

Reply:
yeah, it looks better. I left **'a'** small intentionally 😃